wpilibc/src/main/native/cpp/Tracer.cpp

```diff
@@ -16,6 +16,26 @@ Tracer::Tracer() {
   ResetTimer();
 }
 
+void Tracer::PublishToNetworkTables(std::string_view topicName) {
+  if (!m_publishNT) {
```

Comment:
Condition needs to be flipped
Same goes for DataLog publishing

Java is okay though